src/components/WebexSettings/WebexSettings.scss

```diff
@@ -1,2 +1,14 @@
 @import 'Select';
 @import 'Tabs';
+
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-settings {
+  background-color: var(--wxc-modal-background);
```

Comment:
modal-background must be set by the modal component, this component is just the content of the modal and it can be displayed in other contexts not only in a modal